server/server.ts

```diff
@@ -22,17 +21,17 @@ if (process.env.HTTPS) {
 } else {
   server = new http.Server(app);
 }
-const io = socketIO(server, { origins: '*:*' });
+const io = socketIO(server, { origins: "*:*" });
```

Comment:
single quotes? You can use `npm run prettier` to format according to the current rules